src/daffodil.ts

```diff
@@ -54,7 +54,14 @@ export interface BuildInfo {
   scalaVersion: string
 }
 
-export const errorEvent = 'daffodil.error'
+export const errorEvents = {
+  'daffodil.error.unexpected': 'unexpected error',
```

Comment:
These are aborts, so really this doesn't need to be this pretty. Just printing the original case class name coming over from the scala code would be fine.